index.html

```diff
@@ -185,9 +185,75 @@ <h2>Skills</h2>
         I'm always interested in hearing about new projects, so if you'd like to
         chat please get in touch.
       </p>
-      <form action="#"></form>
+      <form action="https://formspree.io/f/xknagban" method="post">
+        <fieldset>
```

Comment:
- [ ] I suggest removing the fieldset tag to keep your for simple :recycle:

Reply:
Hello @GedeonTS thank you so much for this review and I really appreciate your remarks. However, I was thinking that since I needed to align input elements using a display of flex they should at least be contained within a container. I thought of a div but then again, I realized that <fieldset> would work great since its within a form. Tell me what you think and I will really apreciate your feedback.

Reply:
- Hello @Adholah96 . You did a good job. It's ok to go beyond and explore, and I like your creativity in using tags that are not commonly used in `HTML` such as the `fieldset` tag. Based on the fact that groups your sections and has the same visual effect needed, you're good to go. Good job 👏